nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -739,19 +739,33 @@ define([
                             expectedFiles.add(f);
                         }
                     });
-                    return BulkImportUtil.getMissingFiles(Array.from(expectedFiles)).catch(
-                        (error) => {
-                            // if the missing files call fails, just continue.
-                            console.error(
-                                'Unable to fetch missing files from the Staging Service',
-                                error
+                    return expectedFiles;
+                })
+                .then((expectedFiles) => {
+                    if (
```

Comment:
Split this Promise apart a little bit.
If we're not in an editing state, just return the current state of parameter readiness. Otherwise, go through the evaluation and return that.